lib/plug/conn.ex

```diff
@@ -126,7 +126,8 @@ defmodule Plug.Conn do
   @type assigns :: %{optional(atom) => any}
   @type before_send :: [(t -> t)]
   @type body :: iodata
-  @type cookies :: %{optional(binary) => binary}
+  @type req_cookies :: %{optional(binary) => binary}
+  @type cookies :: %{optional(binary) => term}
```

Comment:
Now `conn.cookies` can also contain terms beyond strings as values, but those are only loaded if explicitly marked as so on the `fetch_cookie` call.